packages/page-accounts/src/Accounts/modals/Qr.tsx

```diff
@@ -36,7 +36,7 @@ function QrModal ({ className = '', onClose, onStatusChange }: Props): React.Rea
   const password2StateHooks = useState(defaultPasswordState);
   const [{ isPassValid: isPass2Valid }] = password2StateHooks;
 
-  const isValid = !!address && isNameValid && isPassValid && isPass2Valid;
+  const isValid = isAddress ? (!!address && isNameValid) : (!!address && isNameValid && isPassValid && isPass2Valid);
```

Comment:
```suggestion
  const isValid = isAddress
    ? (!!address && isNameValid)
    : (!!address && isNameValid && isPassValid && isPass2Valid);
```

Getting a bit too much to read on a single line (should probably also remove the duplication there now)